src/main/resources/reference.conf

```diff
@@ -12,10 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+akka-persistence-jdbc.migration.v4.journal.rows-per-transaction = 100
+akka-persistence-jdbc.migration.v4.snapshot.rows-per-transaction = 100
```

Comment:
This is duplicated with the configuration below, so these can be removed

Reply:
I thought it would be convenient to limit rows per transaction for journal and snapshots as the average size of event and snapshot might be different, so speed of migration can be different.

Reply:
I agree. However what I meant to say what these configuration settings are also set on line 22 and 24, so there is no need to configure this twice

Reply:
ah you're right, committed by accident :)